src/fetchers/top-languages-fetcher.js

```diff
@@ -47,6 +53,13 @@ async function fetchTopLanguages(username, langsCount = 5, exclude_repo = []) {
   }
 
   let repoNodes = res.data.data.user.repositories.nodes;
+
+  if (!include_forks) {
+    repoNodes = repoNodes.filter((isFork) => {
+      return !isFork.isFork;
+    });
+  }
```

Comment:
```suggestion
  if (!include_forks {
    repoNodes = repoNodes.filter(({ isFork }) => !isFork);
  }
```

I think desctructuring would simplify filter callback : )

Reply:
Will fix in the TS version after https://github.com/anuraghazra/github-readme-stats/pull/832 is merged.

Reply:
Great! :)